.gitignore

```diff
@@ -0,0 +1,33 @@
+### C++ ###
```

Comment:
I think your `a.out` binary is still in the Git repo. - you need to delete it and commit